src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 33px 17px 17px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  box-sizing: border-box;
+
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: $card-hover;
+    }
+
+    .card__btn-buy {
+      background-color: $white-color;
+      color: $blue-color;
+      border: solid 1px $blue-color;
+    }
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code {
+    color: $gray-color;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+  }
```

Comment:
Font size and weight for `.card__product-code` are set explicitly here, but it would be better to use a mixin or extend a placeholder if these styles are repeated across different elements to ensure consistency and easier maintenance.